### ISSUE TITLE
Leave only one`main` landmark on files page

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#43085]
There should be only one main landmark on the page. If there are any reasons to have several then we have to differentiate between them: https://www.w3.org/WAI/ARIA/apg/patterns/landmarks/examples/main.html

In my eyes there is no reason to have several `main` landmarks for files page. On all other pages (which are migrated to vue) there is only `app-content-vue` main landmark, like: 

![Screenshot from 2024-01-24 16-39-03](https://github.com/nextcloud/server/assets/6078378/a5ba06fd-7687-403a-92cc-0cdfd18d9453)

Actually `<main id="content" class="app-files">` should not exist because files page is already migrated to vue.

**Comment by ShGKme** [comment]
There is the same issue in some settings. There we have 2 options:
1. Fully migrated settings, including navigation, into Vue
   - Could be not so easy, because apps register many settings parts
   - Gives as new fancy skip actions
2. Remove `NcContent` from some settings

**Comment by JuliaKirschenheuter** [comment]
> Remove NcContent from some settings

Seems not to be a problem for a moment
